### PR TITLE
Feature/add password

### PR DESCRIPTION
**Pull request by francoisph**

### CODE REVIEW

File: client/src/view/scenic/pages/SettingsTableView.html [hunk 1]
```diff
@@ -44,4 +44,30 @@
             <% } %>
         </div>
     </fieldset>
+    <fieldset>
+        <legend data-i18n>Authentication</legend>
+            <div>
+                <label data-i18n>Enable the Authentication when lauching Scenic </label>
```
Comment (ubald) [inline client/src/view/scenic/pages/SettingsTableView.html]: [...]launching[...]

File: client/src/view/scenic/pages/SettingsTableView.html [hunk 2]
```diff
@@ -44,4 +44,30 @@
...
+                <label class="checkbox" for="checkboxAuth"></label>
+            </div>
+            <% if ( authOn ) { %>
+                <% if ( haveUser ) { %>
+                <button id="resetLogin" class="btn btn-login">Reset password</button>
+                <% } else { %>
+                    <div class="input-group">
+                        <label data-i18n>Create a new password </label>
+                        <div>
+                            <input id="newPassword" name="newPassword" type="newPassword" placeholder="" data-i18n="[placeholder]newPassword">
```
Comment (ubald) [inline client/src/view/scenic/pages/SettingsTableView.html]: Not a valid input type.

Comment (ubald) [inline client/src/view/scenic/pages/SettingsTableView.html]: Placeholder has to be filled at both places. Also, placeholder should be "New Password" not "newPassword".

File: client/src/scss/partials/_forms.scss [hunk 3]
```diff
@@ -115,7 +115,21 @@ input[type=password] {
     border: lighten($backgroundColorDark, 15%) solid 1px;
   }
 }
-
+input[type=newPassword] {
```
Comment (ubald) [inline client/src/scss/partials/_forms.scss]: *newPassword* is not a valid input type. Use *id* or *class* to target.

**Comment by francoisph** [comment]
@ubald review modifications

File: client/src/model/pages/Settings.js [hunk 4]
```diff
@@ -45,7 +49,78 @@ define( [
...
             Page.prototype.initialize.apply( this, arguments );
+
+            this.authentification = new Authentication();
+
+            async.series( [
+
+                callback => {
+                    this.authentification.isEnabled(( result, error ) => {
+                        if ( error ) {
+                            callback(xhr);
```
Comment (ubald) [inline client/src/model/pages/Settings.js]: 'tention au xhr qui traine

File: client/src/model/pages/Settings.js [hunk 5]
```diff
@@ -45,7 +49,78 @@ define( [
...
+                    });
+                },
+
+                callback => {
+                    this.on( 'change:authOn', () => {
+                        this.authentification.enable( this.get( 'authOn' ), ( result, error ) => {
+                            console.log('enable plop ', result)
+                            if ( error ) {
+                                this.scenic.sessionChannel.vent.trigger( 'error', i18n.t( 'Authentication cannot be changed' ));
+                                callback(xhr);
```
Comment (ubald) [inline client/src/model/pages/Settings.js]: xhr

**Pull request merged**